src/components/Form/Form.tsx

```diff
@@ -3,47 +3,89 @@ import {
 	StyledDiv,
 	StyledFieldset,
 	StyledInput,
+	StyledLabel,
 	StyledLegend,
```

Comment:
Łatwiej jest zaimportować to jako import * as S from 'Form.Styled.ts"
Wtedy masz jeden import, jedna linia i mozesz uzywac w JSX np. <S.StyledDiv> <S.StyledFieldset>